systemd/authd.service

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Authd daemon service
+After=authd.socket
+Requires=authd.socket
+PartOf=authd.socket
+
+[Service]
+Type=notify
+ExecStart=/sbin/authd
+
+# Some daemon restrictions
+CapabilityBoundingSet=
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=true
+PrivateDevices=yes
+PrivateMounts=yes
+PrivateNetwork=yes
+PrivateTmp=yes
+ProcSubset=pid
```

Comment:
We are listing every processes under /proc when cleaning up old users from cache, and check their associated uid. I think this will be an issue with this.

Reply:
Thanks for checking! I read the "not directly associated with process management" as with "the current pid", which was a misread from me. Sounds good then!